examples/contract.example.ts

```diff
@@ -143,4 +143,15 @@ const account = {
      */
     const transferMethodCall = await contract.transfer(contract.address, 1, {gasLimit: 300000});
     console.log(transferMethodCall.transactionId);
+
+	/**
+     * Filtering contract events
+     */
+	const filter = {
+        address: contract.address
+    };
+    const fromTimestamp = "1000000000.000000000";
+    const toTimestamp = "1999999999.9999999999";
+    const events = await contract.queryFilter(filter, fromTimestamp, toTimestamp);
```

Comment:
I think that we should support not only this string format for the timestamp but `number` as-well. It can hold `seconds` + `nanos` up until Tuesday, 5 June 2255 23:48:20